### PR TITLE
Feature/sgload cpu profiles

### PR DESCRIPTION
**Pull request by tleyden**
#### Fixes #.

- [x] Ran `flake8` -- [see!](https://gist.github.com/tleyden/3cfdea427efd095a3e28c2b22a42a240)

#### Changes proposed in this pull request:

- When running sgload based perftest, it fetches cpu/mem profiles (was not doing so before)
- Refactor out the yml playbook for setting up the profile collection

### CODE REVIEW

File: libraries/utilities/fetch_sync_gateway_profile.py [hunk 1]
```diff
@@ -21,3 +23,18 @@ def fetch_sync_gateway_profile(cluster_config, folder_name):
         shutil.move("/tmp/sync_gateway_profile", "testsuites/syncgateway/performance/results/{}/".format(folder_name))
 
     print("\n")
+
+
+
```
Comment (sethrosetter) [inline libraries/utilities/fetch_sync_gateway_profile.py]: Did you push the flake8 changes? This looks like it should be caught by that

Comment (tleyden) [inline libraries/utilities/fetch_sync_gateway_profile.py]: I'd forgotten -- just pushed a new commit

**Comment by tleyden** [comment]
@sethrosetter this is ready to merge AFAIK

I need the streaming output changes right now in particular

**Comment by tleyden** [comment]
I have a "smoke test" run in progress against gateload to ensure this PR doesn't break anything.

http://uberjenkins.sc.couchbase.com/view/Performance/job/sync-gateway-perf-test-copy-ro/33/

**Comment by tleyden** [comment]
That smoke test hung and never completed.  Re-assigning this PR to myself to review and figure out what's wrong

**Comment by sethrosetter** [comment]
@tleyden Can you rebase?

**Comment by tleyden** [comment]
Closing this -- see https://github.com/couchbaselabs/mobile-testkit/pull/1007

**Pull request closed without merging**